### PR TITLE
Add faces for merlin-mode

### PR DESCRIPTION
**Pull request by arichiardi**
Add zenburn colors for `merlin-mode`.

### CODE REVIEW

File: zenburn-theme.el [hunk 1]
```diff
@@ -1318,6 +1318,19 @@ Also bind `class' to ((class color) (min-colors 89))."
 ;;;;; yascroll
    `(yascroll:thumb-text-area ((t (:background ,zenburn-bg-1))))
    `(yascroll:thumb-fringe ((t (:background ,zenburn-bg-1 :foreground ,zenburn-bg-1))))
+
+;;;;; merlin
```
Comment (bbatsov) [inline zenburn-theme.el]: Sections are sorted alphabetically in the source code.

Comment (arichiardi) [inline zenburn-theme.el]: Oh, l'll change

File: zenburn-theme.el [hunk 2]
```diff
@@ -955,6 +955,18 @@ Also bind `class' to ((class color) (min-colors 89))."
    `(nav-face-hdir ((t (:foreground ,zenburn-red))))
    `(nav-face-file ((t (:foreground ,zenburn-fg))))
    `(nav-face-hfile ((t (:foreground ,zenburn-red-4))))
+;;;;; merlin
```
Comment (bbatsov) [inline zenburn-theme.el]: This should be right after `magit`. N's after M in the alphabet ;-)

Comment (arichiardi) [inline zenburn-theme.el]: As you can see there was an error already in the list. And mine is after N already. Will fix the previous error too then.

**Pull request merged**